dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -468,7 +469,7 @@ public void render_spanish_contentlets_on_english_page(
                         .setPageMode(PageMode.LIVE)
                         .build(),
                 mockRequest, mockResponse);
-        Assert.assertTrue("ESP = "+html , html.contains("content2Spa") && html.contains("content3Spa"));
+        assertTrue("ESP = "+html , html.contains("content2Spa") && html.contains("content3Spa"));
```

Comment:
Codacy found an issue: [The String literal "ESP = " appears 5 times in this file; the first occurrence is on line 472](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6309437)